docs/DeveloperGuide.md

```diff
@@ -433,24 +429,23 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-**Use case: Add a remark**
+**Use case: Add a Remark to a Client**
 
 **MSS**
 
-1. User requests to create a Mcdonalds client
-2. JeeqTracker creates and adds a client called McDonalds into JeeqTracker
-3. User requests to add Justin as a Remark to Mcdonalds
-4. JeeqTracker adds Justin and his details into the Mcdonalds as the Remark
+1. User requests to list all clients
+2. JeeqTracker shows a list of clients
+3. User requests to add Punctual Buyer as a Remark to client Alice
+4. JeeqTracker adds Punctual Buyer as the Remark to Alice's contact card
 
     Use case ends.
 
 **Extensions**
 
-* 1a. Client Mcdonalds already exists.
-    * 1a1. JeeqTracker shows an error message.
+* 2a. The list is empty.
 
-        Use case resumes at 1.
-* 3a. User fails to provide all necessary details to create a Remark.
+  Use case ends.
+* 3a. User fails to provide a valid command format to create a Remark.
     * 3a1. JeeqTracker shows an error message.
```

Comment:
Your code looks great! Perhaps instead of stating the list is empty as a extension, stating the precondition that JeeqTracker is empty can be better as the state of JeeqTracker can be known at the start?

Reply:
Good idea!